.packit.yml

```
@@ -35,15 +35,14 @@ jobs:
       - fedora-all
     specfile_path: bpfman.spec
   - job: copr_build
-    trigger: release
-    branch: main
+    trigger: pull_request
+    targets: [fedora-stable]
```

nit, I'd format this in a different way. IMHO it enhances readability:
```suggestion
    targets:
      - fedora-stable
```

---

Besides that, I'd be interested in getting to know if we're targeting any specific arch.